components/textfield/stories/textfield.stories.js

```diff
@@ -163,13 +198,90 @@ export default {
 	},
 };
 
-export const Default = Template.bind({});
-Default.args = {};
+const TextFieldGroup = ({
+	...args
+}) => {
+	return html`
+		<div style="display: flex; flex-direction: column; gap: 2rem;">
+			${Template({
+				...args
+			})}
+			${isChromatic() ?
```

Comment:
Can you update these references to `window.isChromatic()` now that the toggle has merged?